src/main/integrations/electron-context.ts

```diff
@@ -28,7 +28,16 @@ export const electronContextIntegration = defineIntegration(() => {
         delete event.request.headers['User-Agent'];
       }
 
-      const { release = getDefaultReleaseName(), environment = getDefaultEnvironment() } = client.getOptions();
+      const {
+        release = getDefaultReleaseName(),
+        environment = getDefaultEnvironment(),
+        sendDefaultPii,
+      } = client.getOptions();
+
+      if (sendDefaultPii !== false && typeof event.user?.ip_address === 'undefined') {
```

Comment:
Could we maybe make this more explicit by `sendDefaultPii === true`?

Reply:
> A user might opt-out from this behavior by configuring the SDK with send_default_pii: false.

I read this and assumed that `{{auto}}` would be added unless `sendDefaultPii` was explicitly set to false but now I've checked and the default is false!

Reply:
Thanks, reworded the description in the issue to be less confusing 😬